core/bootstrap.php

```diff
@@ -37,6 +37,7 @@
 @date_default_timezone_set('UTC');
 
 disableEaccelerator();
+printTestDoxHint();
```

Comment:
Seems that currently printed in any case. Shouldn't that only be shown when running tests and if testdox isn't enabled?

Reply:
@sgiehl I can add this, but given testdox will rarely be used, and will only be set manually, I didn't think it was worth the added complexity. Let me know if this is still an important change.